composer.json

```diff
@@ -11,7 +11,8 @@
     "require": {
         "php": ">=5.5.0",
         "ext-curl": "*",
-        "ext-json": "*"
+        "ext-json": "*",
+        "vlucas/phpdotenv": "^3.3"
```

Comment:
We should not include this library, as this just increases the dependency count of our package and will conflict with other versions of the package.